app/assets/stylesheets/fae/modules/forms/_colorpicker.scss

```diff
@@ -0,0 +1,57 @@
+.js-color-picker {
+  width: 180px !important; // overrides input.string input
+}
+
+div.cp-color-picker {
+  border: 1px solid $c-darker-grey;
+  padding: 10px 10px 0;
+  background: #eee;
+  overflow: visible;
+  border-radius: 3px;
+
+  .cp-alpha {
+    margin: 10px 0;
+    height: 6px;
+    border-radius: 6px;
+    overflow: visible;
+    border: 1px solid $c-darker-grey;
+    box-sizing: border-box;
+    background: linear-gradient(to right, rgba(238, 238, 238, 1) 0%, rgba(238, 238, 238, 0) 100%);
```

Comment:
Scss can convert hex to RGB naturally, so these should also use variables: `rgba($c-lightest-grey, 1)` and `rgba($c-lightest-grey, 0)`